src/embed/types.rs

```diff
@@ -44,8 +52,12 @@ pub struct CloseTransactionRequest {
 #[derive(SerJson)]
 pub struct CloseTransactionResponse {}
 
-#[derive(DeJson)]
-pub struct GetRootRequest {}
+#[derive(Debug, DeJson, SerJson)]
+pub struct GetRootRequest {
+    #[nserde(rename = "headName")]
+    #[nserde(skip_serializing_if = "Option::is_none")]
+    pub head_name: Option<String>,
```

Comment:
What brought on this change? replicache-client doesn't support this field:

https://github.com/rocicorp/replicache-client/blob/master/repm/types.go#L14

Reply:
huh, guessing i wanted to use it in a dispatch test but then something happened and i removed the use of it. someone later added it to the rpc. can remove it if you want tho seems useful for poking at things.